src/textual/app.py

```diff
@@ -348,14 +352,17 @@ def __init__(
         css_path = css_path or self.CSS_PATH
         if css_path is not None:
             # When value(s) are supplied for CSS_PATH, we normalise them to a list of Paths.
+            css_paths: List[PurePath]
             if isinstance(css_path, str):
                 css_paths = [Path(css_path)]
             elif isinstance(css_path, PurePath):
                 css_paths = [css_path]
             elif isinstance(css_path, list):
                 css_paths = []
                 for path in css_path:
-                    css_paths.append(Path(path) if isinstance(path, str) else path)
+                    css_paths.append(
```

Comment:
I wonder if some of the code around here could be simplified a wee bit given that it's safe to `Path` a `Path`.